docfx_yaml/extension.py

```diff
@@ -845,9 +845,13 @@ def _update_friendly_package_name(path):
                         # Find the index of the current default value argument
                         index = len(args) + count - offset_count
 
-                        # Only add defaultValue when str(default) doesn't contain object address string(object at 0x)
-                        # inspect.getargspec method will return wrong defaults which contain object address for some default values, like sys.stdout
-                        if 'object at 0x' not in str(default):
+                        # Only add defaultValue when str(default) doesn't
+                        # contain object address string, for example:
+                        # (object at 0x) or <lambda> at 0x7fed4d57b5e0
+                        # inspect.getargspec method will return wrong defaults
+                        # which contain object address for some default values,
+                        # like sys.stdout.
```

Comment:
I feel like there is a missing period or "The" or something here. I don't understand this comment.

Reply:
Updated the docstring!